src/components/dialog/CreateTemplateDialog.vue

```diff
@@ -0,0 +1,118 @@
+<template>
```

Comment:
is this component used anywhere?
the content looks weird, it seems to be a mix of the create component stuff and the create issue template stuff